src/toil/common.py

```diff
@@ -543,7 +543,7 @@ def _addOptions(addGroupFn, config):
     #Debug options
     #
     addOptionFn = addGroupFn("toil debug options", "Debug options")
-    addOptionFn("--debug-worker", default=False, action="store_true", dest="debugWorker",
+    addOptionFn("--debugWorker", default=False, action="store_true",
```

Comment:
Why this change? is this because of a bad merge or something? This line got changed 3 times in 4 commits! And the change has already been applied on master. IMO this PR should be squashed or otherwise rebased so this line won't be changed.

Reply:
Yes, it was a bad merge.  I'll just squash this and make a new one.